compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -786,6 +786,12 @@ OMR::SymbolReferenceTable::findOrCreateNewObjectNoZeroInitSymbolRef(TR::Resolved
    return findOrCreateRuntimeHelper(TR_newObjectNoZeroInit, true, true, true);
    }
 
+TR::SymbolReference *
+OMR::SymbolReferenceTable::findOrCreateAcmpHelperSymbolRef(TR::ResolvedMethodSymbol *)
+   {
+   return findOrCreateRuntimeHelper(TR_acmpHelper, true, true, true);
+   }
+
```

Comment:
The helper references have to live in OMR because of the unfortunate way things are architected at the moment.  But does the symbol reference creation need to live here?